### PR TITLE
OKTA-807236 - Remove rate limit content

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- as per JIRA ticket, removing content that is no longer relevant, as per PM
- **Is this PR related to a Monolith release?** n/a
### Resolves:

* [OKTA-807236](https://oktainc.atlassian.net/browse/OKTA-807236)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rate-limits/index.md](https://github.com/okta/okta-developer-docs/blob/8582600beecad2f42f4786591e7912ae7e76d914/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frate-limits%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/14f85bc1-37ea-43b9-9061-ee5b0cbb0746) | 66  | 85  | 50  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md](https://github.com/okta/okta-developer-docs/blob/8582600beecad2f42f4786591e7912ae7e76d914/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-additional-limits%2Findex.md) | [65](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7d2ced76-3a77-4fc8-813d-4f1e37c46c12) | 42  | 63  | 39  | 89  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md](https://github.com/okta/okta-developer-docs/blob/8582600beecad2f42f4786591e7912ae7e76d914/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-enduser%2Findex.md) | [73](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f605bc0d-07f0-4e08-838f-d4bdf448c679) | 23  | 69  | 87  | 93  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md](https://github.com/okta/okta-developer-docs/blob/8582600beecad2f42f4786591e7912ae7e76d914/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-mgmt%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7bc54260-f19e-48b7-81dd-f2fef85121a3) | 81  | 68  | 81  | 90  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.94108a14-fe68-4362-91bb-9b96b6c2ff92?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rate-limits/index.md](https://github.com/okta/okta-developer-docs/blob/239858eed92bb913f9064ad0e351766d79ec381c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frate-limits%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4c70a965-5b35-45ae-8f90-38011abc5959) | 66  | 85  | 50  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md](https://github.com/okta/okta-developer-docs/blob/239858eed92bb913f9064ad0e351766d79ec381c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-additional-limits%2Findex.md) | [65](https://okta.acrolinx.cloud/api/v1/checking/scorecards/73368e80-45d7-4353-97cb-2fee45c67a51) | 42  | 63  | 39  | 89  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md](https://github.com/okta/okta-developer-docs/blob/239858eed92bb913f9064ad0e351766d79ec381c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-enduser%2Findex.md) | [73](https://okta.acrolinx.cloud/api/v1/checking/scorecards/77566e9b-b08c-4d85-850e-b44917243caf) | 23  | 69  | 87  | 93  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md](https://github.com/okta/okta-developer-docs/blob/239858eed92bb913f9064ad0e351766d79ec381c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-mgmt%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c99033d0-e2f5-41eb-b222-9c8479473969) | 81  | 68  | 81  | 90  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.ff6fafa0-2f2b-4e5f-af89-14efc3ca6e2b?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rate-limits/index.md](https://github.com/okta/okta-developer-docs/blob/729cb159a318657776709cd707e9aafb56d3c53a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frate-limits%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/76152d52-667a-466c-8183-acbcc0c4f250) | 66  | 85  | 51  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md](https://github.com/okta/okta-developer-docs/blob/729cb159a318657776709cd707e9aafb56d3c53a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-additional-limits%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/23142e83-4642-4830-bd1b-a04d350f2eab) | 59  | 92  | 56  | 96  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md](https://github.com/okta/okta-developer-docs/blob/729cb159a318657776709cd707e9aafb56d3c53a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-enduser%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b83cf35c-b795-4432-8b21-fe880256bba4) | 24  | 87  | 100  | 93  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md](https://github.com/okta/okta-developer-docs/blob/729cb159a318657776709cd707e9aafb56d3c53a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-mgmt%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/75e7434b-6c8e-44c0-8036-7c5ae7af7403) | 81  | 68  | 81  | 90  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.e655dc0e-aaf4-4235-8cd9-52a986e77ff3?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rate-limits/index.md](https://github.com/okta/okta-developer-docs/blob/c74c61c1f2c20e4a90bc37eaa697ab0794707aa3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frate-limits%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6634dd6f-b251-4127-8fa9-1c6936842c8c) | 66  | 85  | 51  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md](https://github.com/okta/okta-developer-docs/blob/c74c61c1f2c20e4a90bc37eaa697ab0794707aa3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-additional-limits%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7c4b9424-ae87-402f-804f-3d5ee4430f43) | 59  | 92  | 56  | 96  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md](https://github.com/okta/okta-developer-docs/blob/c74c61c1f2c20e4a90bc37eaa697ab0794707aa3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-enduser%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/97702020-2607-4970-b80d-7a2e1beb8dbe) | 24  | 87  | 100  | 93  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md](https://github.com/okta/okta-developer-docs/blob/c74c61c1f2c20e4a90bc37eaa697ab0794707aa3/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-mgmt%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4a089eb9-4ad6-4f51-be3d-8eb40bc41bc2) | 81  | 68  | 81  | 90  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.fe45d68f-497e-48f8-81f0-5ce2440388ed?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md [hunk 1]
```diff
@@ -6,12 +6,12 @@ excerpt: >-
 
 # Additional rate limits
 
-This page provides Okta's additional limits on:
+This page provides additional rate limits on:
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:9]:
```suggestion
This page provides additional rate limits:
```

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md [hunk 2]
```diff
@@ -105,13 +102,13 @@ API calls to the Org creator endpoint (POST `/api/v1/orgs`) have a limit of 10 o
 
 ## Workforce license rate limit multiplier
 
-Workforce orgs that are created after January 7, 2021 have increased default rate limits. This increase is for [specific endpoints](#list-of-endpoints) and depends on a Workforce org's license count (Universal Directory or Single-Sign On).
+Workforce orgs that are created after January 7, 2021 have increased default rate limits. This increase is for [specific endpoints](#list-of-endpoints) and depends on a Workforce org's license count (Universal Directory or single-sign-on).
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:105]:
```suggestion
Workforce orgs that are created after January 7, 2021 have increased default rate limits. This increase is for [specific endpoints](#list-of-endpoints) and depends on a Workforce org's license count (Universal Directory or Single Sign-On).
```

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 3]
```diff
@@ -37,20 +36,21 @@ See the following list of per-minute limits. If an endpoint isn't in this list,
...
 | `/login/token/redirect`<br>Eligible for dynamic scale and workforce multiplier                                                                                                             | 100              | 600              | 600     | 600        | 600                   |
-| <ApiLifecycle access="ie" /> **Identity Engine APIs:**</br>Identity Engine rate limits are configured to support 1000 Identity Engine authentication flows per minute. That is, depending on authentication flows, some endpoint limits may differ.                                                                                                        |
+| <ApiLifecycle access="ie" /> **Identity Engine APIs:**</br> Identity Engine rate limits are configured to support 1000 Identity Engine authentication flows per minute. That is, depending on authentication flows, some endpoint limits may differ.                                                                                                        |
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:39]:
```suggestion
| <ApiLifecycle access="ie" /> **Identity Engine APIs:**</br> Identity Engine rate limits are configured to support 1000 Identity Engine authentication flows per minute. Depending on the authentication flow, some endpoint limits may differ.                                                                                                        |
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/openidconnect/prep.md](https://github.com/okta/okta-developer-docs/blob/4da81c439b44984d508b380ff1c81e566f5b684d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-sso-integration%2Fmain%2Fopenidconnect%2Fprep.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9d53d173-82f8-4049-a2b5-2831052c4ba1) | 56  | 82  | 68  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rate-limits/index.md](https://github.com/okta/okta-developer-docs/blob/4da81c439b44984d508b380ff1c81e566f5b684d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frate-limits%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9403a6be-7b3d-4701-9453-cb19ba60efd1) | 64  | 85  | 51  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md](https://github.com/okta/okta-developer-docs/blob/4da81c439b44984d508b380ff1c81e566f5b684d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-additional-limits%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/62875723-40c4-4da1-a4e3-506f701f72a8) | 59  | 92  | 56  | 96  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md](https://github.com/okta/okta-developer-docs/blob/4da81c439b44984d508b380ff1c81e566f5b684d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-enduser%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9ecd390e-eef4-4418-ad6f-85c6a85843c0) | 24  | 87  | 100  | 93  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md](https://github.com/okta/okta-developer-docs/blob/4da81c439b44984d508b380ff1c81e566f5b684d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-mgmt%2Findex.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f610b151-d03b-41a6-a899-33ace522ede9) | 52  | 68  | 81  | 90  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.80abca5a-cb27-4d45-bbcd-be68ac9005bd?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/openidconnect/prep.md](https://github.com/okta/okta-developer-docs/blob/7e01b13ecf32bee322224d0cea37d2b8f9cce6e6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-sso-integration%2Fmain%2Fopenidconnect%2Fprep.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0d35363a-96f6-431a-8af3-4f764b4f9307) | 56  | 82  | 68  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rate-limits/index.md](https://github.com/okta/okta-developer-docs/blob/7e01b13ecf32bee322224d0cea37d2b8f9cce6e6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frate-limits%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d0d95875-1c60-4b6b-9d79-a366c73e99e4) | 64  | 85  | 51  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md](https://github.com/okta/okta-developer-docs/blob/7e01b13ecf32bee322224d0cea37d2b8f9cce6e6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-additional-limits%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9214f23f-14a4-4d64-b97f-560cdff3fa33) | 59  | 92  | 56  | 96  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md](https://github.com/okta/okta-developer-docs/blob/7e01b13ecf32bee322224d0cea37d2b8f9cce6e6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-enduser%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f5d2cc1f-4709-4e9b-b250-b92c502110bc) | 24  | 87  | 100  | 93  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md](https://github.com/okta/okta-developer-docs/blob/7e01b13ecf32bee322224d0cea37d2b8f9cce6e6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-mgmt%2Findex.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/824fe3a2-ca3d-4afb-a641-61ecab7e555a) | 52  | 68  | 81  | 90  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.181121fa-db97-463e-9715-0844cc25aa74?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md (hunk 2 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md:105]: FYI ... I think this was an Acrolinx fix; hopefully won't fail me :o)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/openidconnect/prep.md](https://github.com/okta/okta-developer-docs/blob/da04902edb0e0e485cf2f73fe40ad11a04ba074f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-sso-integration%2Fmain%2Fopenidconnect%2Fprep.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fc67f1b9-7189-4ce5-9789-5cd23171eeac) | 56  | 82  | 68  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rate-limits/index.md](https://github.com/okta/okta-developer-docs/blob/da04902edb0e0e485cf2f73fe40ad11a04ba074f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frate-limits%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9310fe2d-bb94-4121-9a73-0dca5a6ba97b) | 64  | 85  | 51  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md](https://github.com/okta/okta-developer-docs/blob/da04902edb0e0e485cf2f73fe40ad11a04ba074f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-additional-limits%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8d671a5d-739c-4ad2-9120-f90d0fbc5045) | 56  | 92  | 56  | 96  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md](https://github.com/okta/okta-developer-docs/blob/da04902edb0e0e485cf2f73fe40ad11a04ba074f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-enduser%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/04cba7b1-b125-46c3-8596-7814313bdc04) | 24  | 87  | 100  | 93  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md](https://github.com/okta/okta-developer-docs/blob/da04902edb0e0e485cf2f73fe40ad11a04ba074f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-mgmt%2Findex.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/de3ae300-727a-4e19-a597-52646d335a88) | 52  | 68  | 81  | 90  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.3e846a23-765a-49ef-ac87-80f67e55c20a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/openidconnect/prep.md](https://github.com/okta/okta-developer-docs/blob/0359a59de366e3245aaee98105113f9bf8f359d6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fbuild-sso-integration%2Fmain%2Fopenidconnect%2Fprep.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e87ee016-ff22-4672-86c7-50a17909f0ba) | 56  | 82  | 68  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rate-limits/index.md](https://github.com/okta/okta-developer-docs/blob/0359a59de366e3245aaee98105113f9bf8f359d6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frate-limits%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/80d4ec58-997a-4944-b79e-5231286f7103) | 64  | 85  | 51  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md](https://github.com/okta/okta-developer-docs/blob/0359a59de366e3245aaee98105113f9bf8f359d6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-additional-limits%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d17a3416-d860-4eb4-9037-cf101d703bf6) | 56  | 92  | 56  | 96  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md](https://github.com/okta/okta-developer-docs/blob/0359a59de366e3245aaee98105113f9bf8f359d6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-enduser%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b5c7578d-d439-4d83-ae46-c3fe0ecefc50) | 24  | 87  | 100  | 93  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md](https://github.com/okta/okta-developer-docs/blob/0359a59de366e3245aaee98105113f9bf8f359d6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frl-global-mgmt%2Findex.md) | [90](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b487633b-e99b-417c-b965-763054fdb8e9) | 81  | 81  | 100  | 90  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.24b8f44a-7c62-46e1-8d8c-d380d84de8da?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**